src/utils/utils_posix_concurrency.c

```diff
@@ -70,3 +71,56 @@ int utils_read_unlock(utils_rwlock_t *rwlock) {
 int utils_write_unlock(utils_rwlock_t *rwlock) {
     return pthread_rwlock_unlock((pthread_rwlock_t *)rwlock);
 }
+
+void utils_atomic_load_acquire_u64(uint64_t *ptr, uint64_t *out) {
+    utils_annotate_acquire(ptr);
```

Comment:
Could you fix the annotations in this PR, as I did in #1138 ?

Reply:
ok I will check it